### PR TITLE
Initial page on Prop J

### PR DESCRIPTION
**Pull request by benhamill**

### CODE REVIEW

File: _posts/2018-10-09-proposition-j.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title: Proposition A
```
Comment (h3h) [inline _posts/2018-10-09-proposition-j.md]: J

File: _posts/2018-10-09-proposition-j.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+---
+
+Proposition J is about requiring any "comprehensive rewrite" of the city land
```
Comment (h3h) [inline _posts/2018-10-09-proposition-j.md]: Please do use proper “curly quotes” everywhere.

Comment (h3h) [reply _posts/2018-10-09-proposition-j.md]: Hmm. I don’t think so.

Comment (benhamill) [reply _posts/2018-10-09-proposition-j.md]: I just checked it out locally and they looked like smart quotes to me:

> Proposition J is about requiring any “comprehensive rewrite” of the…

I can still change it if you like. 🤷🏻‍♂️

**Comment by benhamill** [comment]
I busted this down to just the sketch we talked about today and made another branch from it with the other commentary I'd added that we can flesh out and deploy later on.

**Pull request merged**